src/Cli/dotnet/Commands/Run/CommonRunHelpers.cs

```diff
@@ -26,4 +26,20 @@ public static string GetPropertiesLaunchSettingsPath(string directoryPath, strin
 
     public static string GetFlatLaunchSettingsPath(string directoryPath, string projectNameWithoutExtension)
         => Path.Join(directoryPath, $"{projectNameWithoutExtension}.run.json");
+
+    /// <summary>
+    /// Creates a TerminalLogger or ConsoleLogger based on the provided MSBuild arguments.
+    /// If the environment is detected to be an LLM environment, the logger is adjusted to
+    /// better suit that environment.
+    /// </summary>
+    /// <param name="msbuildArgs"></param>
```

Comment:
Nit: These empty XML doc elements serve no purpose and are a source of bloat.  IMO they should be removed.  There is precedence to not include these empty elements.  This comment applies to multiple locations.